hangman.js

```diff
@@ -1 +1,44 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var word;
+var guesses = [];
+var guessLimit = 7;//counts the guesses//
+
+//split the word into an array of letters, that are shown as dashes until guessed correctly//
+var separateWord = function() {
+  word= prompt("Enter a word for your player to guess!"),toLowerCase(); //ask for word//
+  word = word.split("");
+  var placeholders = "" ;
+    for(var i = 0; i < word.length; i+=1) {
+      letters[i]= "_"
+    }
+  }
```

Comment:
You have an extra curly bracket here that closes out the function. Because of that, the return statement is processed as existing outside of the function (which is why you might have gotten that `Illegal return statement` error.